content/patterns/button/examples/button_idl.html

```diff
@@ -155,36 +154,36 @@ <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
               <td><code>a</code></td>
               <td>
                 <ul>
-                  <li>Set in javascript with <code>button.ariaPressed = 'false';</code>.</li>
-                  <li>Identifies the button as a toggle button.</li>
-                  <li>Indicates the toggle button is not pressed.</li>
+                  <li>JavaScript で <code>button.ariaPressed = 'false';</code> と設定します。</li>
+                  <li>この設定は、ボタンをトグルボタンとして識別します。</li>
+                  <li>トグルボタンが押されていないことを示します。</li>
```

Comment:
言い切りの形にしつつ、
ひとつめのliの「Set」は過去分詞系だとおもいました。
ふたつめのliは、「この設定は」という謎の節が生み出されていたのでトリました。

```suggestion
                  <li>JavaScript にて <code>button.ariaPressed = 'false';</code> によって設定される。</li>
                  <li>ボタンをトグルボタンとして識別する。</li>
                  <li>トグルボタンが押されていないことを示す。</li>
```